packages/@azure/servicebus/data-plane/test/renewLock.spec.ts

```diff
@@ -10,176 +10,1061 @@ chai.use(chaiAsPromised);
-      throw new Error("Define QUEUE_NAME in your environment before running integration tests.");
-    }
-    if (!process.env.SUBSCRIPTION_NAME) {
-      throw new Error(
-        "Define SUBSCRIPTION_NAME in your environment before running integration tests."
-      );
-    }
+  BASIC_SERVICEBUS_CONNECTION_STRING=
+  BASIC_QUEUE_PARTITION=partitioned-queue
+  BASIC_QUEUE=unpartitioned-queue
 
-    namespace = Namespace.createFromConnectionString(process.env.SERVICEBUS_CONNECTION_STRING);
-    queueClient = namespace.createQueueClient(process.env.QUEUE_NAME);
-    topicClient = namespace.createTopicClient(process.env.TOPIC_NAME);
-    subscriptionClient = namespace.createSubscriptionClient(
-      process.env.TOPIC_NAME,
-      process.env.SUBSCRIPTION_NAME
-    );
+*/
 
-    const peekedQueueMsg = await queueClient.peek();
-    if (peekedQueueMsg.length) {
-      throw new Error("Please use an empty queue for integration testing");
-    }
+describe("Premium", function(): void {
+  const PREMIUM_SERVICEBUS_CONNECTION_STRING = check(
+    process.env.PREMIUM_SERVICEBUS_CONNECTION_STRING,
+    "PREMIUM_SERVICEBUS_CONNECTION_STRING"
+  );
+  const namespace = Namespace.createFromConnectionString(PREMIUM_SERVICEBUS_CONNECTION_STRING);
 
-    const peekedSubscriptionMsg = await subscriptionClient.peek();
-    if (peekedSubscriptionMsg.length) {
-      throw new Error("Please use an empty Subscription for integration testing");
-    }
+  const PREMIUM_QUEUE = check(process.env.PREMIUM_QUEUE, "PREMIUM_QUEUE");
+  describe("Unpartitioned Queues", function(): void {
+    const senderClient = namespace.createQueueClient(PREMIUM_QUEUE);
+    const receiverClient = senderClient;
+
+    describe("Tests - Lock Renewal - Peeklock Mode", function(): void {
+      beforeEach(async () => {
+        await beforeEachTest();
+      });
+
+      afterEach(async () => {
+        await namespace.close();
+      });
+
+      it(`renewLock() with Batch Receiver resets lock duration each time.`, async function(): Promise<
+        void
+      > {
+        await testBatchReceiverManualLockRenewalHappyCase(senderClient, receiverClient);
+      });
+
+      it(`Receive a msg using Batch Receiver, wait until its lock expires, completing it now results in error`, async function(): Promise<
+        void
+      > {
+        await testBatchReceiverManualLockRenewalErrorOnLockExpiry(senderClient, receiverClient);
+      });
+
+      it("Receives a message using Streaming Receiver renewLock() resets lock duration each time.", async function(): Promise<
+        void
+      > {
+        await testStreamingReceiverManualLockRenewalHappyCase(senderClient, receiverClient);
+      });
+
+      it("Receive a msg using Streaming Receiver, wait until its lock expires, completing it now results in error", async function(): Promise<
+        void
+      > {
+        await testStreamingReceiverManualLockRenewalErrorOnLockExpiry(senderClient, receiverClient);
```

Comment:
Isn't the above test case a subset of what is happening in the below test case?
i.e isn't `testStreamingReceiverManualLockRenewalErrorOnLockExpiry` a subset of `testAutoLockRenewalConfigBehavior` with config as 
```
{
           maxAutoRenewDurationInSeconds: 0,
           delayBeforeAttemptingToCompleteMessageInSeconds: 31,
           willCompleteFail: true
}
```